include/deal.II/lac/block_vector.h

```diff
@@ -517,6 +518,16 @@ namespace internal
   } /* namespace LinearOperator */
 } /* namespace internal */
 
+
+/** mark dealii::BlockVector< Number > as serial vector
```

Comment:
Documentation lines typically start on the second line after the `/**` (but with a space between that an the author name). 

Please also capitalise the word "mark". This goes for the documentation of the other specialisations as well.

Please also adjust the date on this file.